geode-core/src/main/java/org/apache/geode/cache/query/internal/QueryMonitor.java

```diff
@@ -332,12 +331,7 @@ public void setLowMemory(final ScheduledThreadPoolExecutor executor,
         final boolean isLowMemory,
         final long usedBytes,
         final InternalCache cache) {
-      if (cache.isQueryMonitorDisabledForLowMemory()) {
```

Comment:
Will this be new behavior that we can set the low memory even if the query monitor is disabled?

Reply:
I was trying to clean some of the dependencies the InternalResourceManager had with the InternalCache and thought it was safe to remove this call because I thought if it was disabled then getQueryMonitor would return null. But I was wrong. Even though it was disabled for low memory monitoring it could still exist because of MAX_QUERY_EXECUTION_TIME being > 0. And making this change broke ResourceManagerWithQueryMonitorDUnitTest. I just pushed a new revision that fixes this test but instead of always asking the cache if it is disabled, QueryMonitor is now told at construction time.